upload_new_statsd_exporter_version.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+VERSION=${1:?"Missing version number argument (arg 1)"}
+NEXUS_USER=${2:?"Missing Nexus username argument (arg 2)"}
+ARCH=${3:?"Missing architecture argument (arg 3)"}
```

Comment:
Should such a script only upload the binary of a specific architecture? The upload scipt for Vector uploads all architectures.

https://github.com/stackabletech/docker-images/blob/c75007e0a63ea0bb5277b88e8cb6d4648028f8f2/vector/upload_new_vector_version.sh#L7-L10

Reply:
done, thanks!